docs/pages/reference/yaml-reference.md

```diff
@@ -146,6 +146,10 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
       <li><code>FINISHED</code>: The task should launch and exit successfully (zero exit code). If the task fails (nonzero exit code) then it is retried without relaunching the entire pod.</li>
       </ul></div>
 
+    * `essential`
+
+      Marks this task as either "Essential", where relaunching this task results in relaunching all `RUNNING` tasks in the pod instance, or "Non-essential", where relaunching it does not affect other `RUNNING` tasks in the pod instance. By default this value is `true`, such that if this task needs to be relaunched, then all other tasks sharing the same pod instance are relaunched alongside it. This is only a factor in cases where a given pod has multiple `RUNNING` tasks defined.
```

Comment:
I think the description here while technically accurate is a little confusing.  I would have written this form the perspective of failure instead of relaunch.  e.g. `When an essential task fails, all other tasks in the same pod will fail with it.`  The implementation detail that this is accomplished by the Scheduler on relaunch is maybe uninteresting in this document.

Reply:
Done in #1953